### PR TITLE
PCM-1406

### PR DESCRIPTION
**Pull request by jtrantin**
@renujhamtani @ardasgup Please review.

### CODE REVIEW

**Comment by ardasgup** [comment]
Reviewed! However i don't see any check for IE browser, will this be de facto for all views/browsers ?

**Comment by jtrantin** [comment]
I believe me and Renu agreed that since the visual appearance is so close to the existing implementation, we might as well not duplicate code and implement this for all browsers.

File: app/cases/controllers/severitySelect.js [hunk 1]
```diff
@@ -3,15 +3,20 @@ angular.module('RedhatAccess.cases').controller('SeveritySelect', [
...
     	// INIT
-    	$scope.openedDetails = {};
+        $scope.openedDetails = {};
     	angular.forEach($scope.severities, function(severity) {
     		$scope.openedDetails[severity.name] = false;
     	});
 
-        $scope.toggleDetails = function (severity) {
+        $scope.toggleDetails = function (severity, event) {
+            event.stopPropagation(); // we don't want to toggle severity
```
Comment (ardasgup) [inline app/cases/controllers/severitySelect.js]: Also just verify if this particular jquery works for all browsers, in past we have faced issues with some event handlers not working in older versions of IE. :/

**Comment by ardasgup** [comment]
@jtrantin ack

File: app/cases/controllers/severitySelect.js (hunk 1 above)
Comment (jtrantin) [inline app/cases/controllers/severitySelect.js]: Correct, this will not work in IE8.

**Comment by jtrantin** [comment]
@renujhamtani @ardasgup Pushed the fix as described in PCM-1406.

**Comment by renujhamtani** [comment]
Reviewed! Looks good! Waiting for @ardasgup to review

**Comment by renujhamtani** [comment]
@ardasgup Please review even if its merged as deepak and Swapna need to test this on CI

**Comment by ardasgup** [comment]
Reviewed!

**Pull request merged**